packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-msft-word.tsx

```diff
@@ -8,4 +8,5 @@ export default {
     </svg>
   ),
   styles: {},
+  exportedAs: 'word word-color',
```

Comment:
maybe it would be better to use comma as separator here - then it would be clear that there are _two_ icons exported for that, and that it is not just a single icon exported under `word word-color` name